src/Symfony/Component/Translation/Tests/Writer/TranslationWriterTest.php

```diff
@@ -18,7 +18,7 @@
 
 class TranslationWriterTest extends TestCase
 {
-    public function testWriteTranslations()
```

Comment:
The legacy test should be kept with `@group legacy` and `@expectedDeprecation [msg]` phpdoc annotations.